xwiki-platform-core/xwiki-platform-scheduler/xwiki-platform-scheduler-ui/src/main/resources/XWiki/SchedulerJobSheet.xml

```diff
@@ -39,48 +39,58 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity}}
-#if($doc.getObject('XWiki.SchedulerJobClass'))
-#set($job = $doc.getObject("XWiki.SchedulerJobClass"))
-**$services.localization.render('xe.scheduler.job.name')**
+  <content>{{velocity output='false'}}
+#macro (displayJobProperty $obj $propName)
+  #set ($class = $obj.xWikiClass)
+  #set ($prettyName = $class.get($propName).translatedPrettyName)
```

Comment:
It's good that we are using the translatedPrettyName, but this means the old translations key are not used anymore, like xe.scheduler.job.name, xe.scheduler.job.description, xe.scheduler.job.expression, xe.scheduler.job.script. 

If they are not used anymore (needs to be tested: do a grep inside platform and see if they are used somewhere else) we could deprecate them.